Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -636,6 +639,24 @@ namespace AssetProcessor
             return;
         }
 
+        QString absolutePathToFile = jobEntry.GetAbsoluteSourcePath();
+
+        // Write the LFS pointer file related messages to the job log and display it in the job event details UI.
+        AssetProcessor::SetThreadLocalJobId(jobEntry.m_jobRunKey);
+        AssetUtilities::JobLogTraceListener jobLogTraceListener(jobEntry);
+
+        if (IsLfsPointerFile(absolutePathToFile.toUtf8().constData()))
+        {
+            AZ_Error(AssetProcessor::ConsoleChannel, false,
+                "%s is a git large file storage (LFS) file. "
+                "This is a placeholder file used by the git source control system to manage content. "
+                "This issue most happens if you've downloaded all of O3DE as a zip file. "
```

Comment:
Typo: most happens -> most often happens, or usually happens

Reply:
Good catch. Just fixed it.